pinot-controller/src/main/java/org/apache/pinot/controller/util/TableSizeReader.java

```diff
@@ -99,6 +99,13 @@ public TableSizeDetails getTableSizeDetails(@Nonnull String tableName, @Nonnegat
       _controllerMetrics.setValueOfTableGauge(realtimeTableName, ControllerGauge.TABLE_SIZE_PER_REPLICA_ON_SERVER,
           tableSizeDetails._realtimeSegments._estimatedSizeInBytes / _helixResourceManager.getNumReplicas(
               realtimeTableConfig));
+
+      tableSizeDetails._realtimeSegments._segments.values().stream()
```

Comment:
(optional) Suggest using the regular non-stream APIs for the following reasons:
1. This is indeed more compact, but slightly hard to read
2. It has worse performance comparing to regular APIs